### ISSUE TITLE
OAuth login fails when password contains ampersand `&`

### ISSUE THREAD

**Issue by kellystuard** [kellystuard/terraform-provider-tss#3]
### Terraform Version
```
$ terraform -v
Terraform v0.14.9
+ provider registry.terraform.io/kellystuard/tss v10.9.33
```
### Affected Resource(s)
- data.tss_secret_field

### Terraform Configuration Files
```hcl
terraform {
  required_providers {
    tss = {
      source = "kellystuard/tss"
    }
  }
}

provider "tss" {
  username = "<redacted>"
  # password from TSS_PASSWORD
  tenant = "<redacted>"
}

data "tss_secret_field" "test_password" {
  number = 1
  slug   = "password"
}

output "test_password" {
  value = "Password: ${data.tss_secret_field.test_password.value}"
}
```

### Debug Output
```
$ terraform apply

Error: Oauth token response: (400) {"error":"Login failed."}

  on main.tf line 9, in provider "tss":
   9: provider "tss" {
```
### Panic Output
N/A

### Expected Behavior
Login should have succeeded.

### Actual Behavior
Login failed.

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. Create user in Thycotic Secret Server with `&` in password
2. `terraform apply`

### Important Factoids
None

### References
None

**Comment by Explooosion-code** [comment]
Created PR #4

**Comment by kellystuard** [comment]
Published in Release 10.9.33.2 (v10.9.35).
